docs/releasehistory.rst

```diff
@@ -8,6 +8,10 @@ New features
 ------------
 - Added support in ``AbsoluteAlchemicalFactory`` for handling multiple independent alchemical regions (`#438 <https://github.com/choderalab/openmmtools/pull/438>`_).
 
+Bugfixes
+--------
+- The multistate samplers now issue experimental API warnings via `logger.warn()` rather than `warnings.warn()` (#446)
```

Comment:
This should be in sphinx syntax so code font should be surrounded by `` instead of a single one, and the link should be
```
(`#446 <https://github.com/choderalab/openmmtools/pull/446>`_)
```

Reply:
Fixed!